tests/sdn/persistentips.go

```diff
@@ -26,7 +27,7 @@ import (
 	"kubevirt.io/kubevirt/tests/testsuite"
 )
 
-var _ = SIGDescribe("Persistent IPs", func() {
+var _ = SIGDescribe("Persistent IPs", decorators.PersistentIPs, func() {
```

Comment:
can be removed because of SIGDescribe

Reply:
We may want to have some sdn team features running at kind-ovn and others at normal qemu kubevirtci, is do not hurt to have additional labels.

Reply:
Removing label and filtering by sig-sdn